### PR TITLE
WIP: remove dangerous feature: removing projects and issues by project

### PR DESCRIPTION
**Pull request by glensc**
remove such dangerous feature like delete project  that also deletes all issues and relations to the issues.

approved by @balsdorf 

and we discussed, if someone really needs such functionality, a custom script can be made.

### CODE REVIEW

File: lib/eventum/class.issue.php [hunk 1]
```diff
@@ -1802,32 +1801,6 @@ public function addAssociation($issue_id, $associated_id, $usr_id, $link_issues
     }
 
     /**
-     * Method used to remove the issue associations related to a specific issue.
-     *
-     * @param int|array $issue_id The issue ID
-     * @param int $usr_id
-     */
-    public function deleteAssociations($issue_id, $usr_id = null)
```
Comment (balsdorf) [inline lib/eventum/class.issue.php]: Is this the wrong branch you pushed? I thought you were trying to remove Issue::removeByProjects ?

**Comment by glensc** [comment]
[Project::remove()](https://github.com/eventum/eventum/blob/v3.1.3/lib/eventum/class.project.php#L311) is what calls out `Issue::removeByProjects()`, drop that as well and methods that are used by that method?

**Comment by balsdorf** [comment]
Yeah, I'd remove Project::remove().  If anyone complains, they can just archive the project + remove all users from it

**Comment by glensc** [comment]
@balsdorf ready for merge

**Pull request merged**